src/platform/plugins/shared/data_view_management/public/components/delete_data_view_flyout/delete_data_view_flyout_content.tsx

```diff
@@ -209,10 +209,15 @@ export const DeleteModalContent: React.FC<ModalProps> = ({
     <div>
       {showRelationshipsCallout ? (
         <>
-          <EuiCallOut color="danger" iconType="warning" title={relationshipCalloutText} />
+          <EuiCallOut
+            announceOnMount
```

Comment:
```suggestion
            announceOnMount={false}
```
Similar thing, doesn't seem to depend on user interaction.